src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const startPositionX = 0;
+  const startPositionY = 0;
+  const evacuationPositionX = 1400;
+  const evacuationPositionY = 500;
+
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: startPositionX,
+      y: startPositionY,
+    },
+
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
```

Comment:
more descriptive this way
```suggestion
    goForward(steps = 1) {
```